rest/data/aliases.go

```diff
@@ -35,15 +46,58 @@ func (d *DBAliasConnector) CopyProjectAliases(oldProjectId, newProjectId string)
 	return nil
 }
 
+func (d *DBAliasConnector) UpdateProjectAliases(projectId string, aliases []restModel.APIProjectAlias) error {
+	aliasesToUpsert := []model.ProjectAlias{}
+	aliasesToDelete := []string{}
+	catcher := grip.NewBasicCatcher()
+	for _, aliasModel := range aliases {
+		if aliasModel.Delete {
+			aliasesToDelete = append(aliasesToDelete, restModel.FromAPIString(aliasModel.ID))
+		} else {
+			v, err := aliasModel.ToService()
+			catcher.Add(errors.Wrap(err, "problem converting to project variable model"))
+
+			alias, ok := v.(model.ProjectAlias)
+			if !ok {
+				catcher.Add(errors.New("problem converting to project alias"))
+			}
+			alias.ProjectID = projectId
+			aliasesToUpsert = append(aliasesToUpsert, alias)
+		}
+	}
+	errStrs := model.ValidateProjectAliases(aliasesToUpsert, "All Aliases")
+	if len(errStrs) > 0 {
```

Comment:
This isn't necessary, since if `errStrs` has length 0, iterating over it is a noop.